src/main/java/org/example/Main.java

```diff
@@ -1 +1,53 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int M;
+        int N;
+        System.out.println("Enter M: ");
+        M = scanner.nextInt();
+        System.out.println("Enter N: ");
+        N = scanner.nextInt();
+
+        int [][] matrix = new int [M][N];
+
+        for (int i = 0; i < M; i++) {
+            for (int j = 0; j < N; j++) {
+                matrix[i][j] = N * i + j;
+            }
+
+        }
+
+        System.out.println("Початкова матриця");
+        System.out.println("      ");
+        for (int i = 0; i < M; ++i) {
+            for (int j = 0; j < M; ++j) {
```

Comment:
I think there should be N in the line. And the same for next loops. Because in the result I see another size.